composer.json

```diff
@@ -89,7 +89,8 @@
             "src/Functional/Truthy.php",
             "src/Functional/Unique.php",
             "src/Functional/With.php",
-            "src/Functional/Zip.php"
+            "src/Functional/Zip.php",
+            "src/Functional/ZipAll.php"
```

Comment:
`
$ composer install

                                                                               
  [Seld\JsonLint\ParsingException]                                             
  "./composer.json" does not contain valid JSON                                
  Parse error on line 93:                                                      
  ...ipAll.php",        ]    },    "autolo                                     
  ---------------------^                                                       
  Expected one of: 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '[' - It  
   appears you have an extra trailing comma                                    
`